lexicons/fyi/frontpage/richtext/block.json

```diff
@@ -2,6 +2,16 @@
   "lexicon": 1,
   "id": "fyi.frontpage.richtext.block",
   "defs": {
+    "main": {
+      "type": "object",
```

Comment:
Is there a reason for the extra object here? Can't this be a union directly?

Reply:
I don't think so, at least lex-cli errors when you have a union at the top level of main. I'll verify that though.